### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Upload_UploadFolder_ScreenReader:Voiceover announces incorrect name for the "Browse folder(...)" button.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/AzureStorageExplorer#2610]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac; #Benchmark; #A11ySev2; #Azure Tools Storage Explorer;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
MacOS: Catalina 10.15.3

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and hit enter on  "Upload Folder" control.
6. Navigate to browse(...) control using tab key.
7. Observe the Voiceover announcement.

**Actual:**
Voiceover  announces the incorrect name for the "Browse folder(...)" button.Here announces as "Browse files".

**Expected:**
Voiceover  should announces the correct name for the "browse folder(...)" button.It should announces "browse folder" button.

**User Impact:**
Screen reader user get incorrect information regarding the control if it announces the incorrect name for "Browse folder" control. 

Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:
MAS1.3.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by Shivamd1** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/:f:/p/v-subha/EvxHdg1zkN9PknF1uOxnOiQBejpxKYg9n21DydgQ_om0ew?e=WK2Zih) link with name "2610_A11y_AzureToolsStorageExplorer_Upload_UploadFolder_ScreenReader" please refer the video for reference.
Please let us know if there is any problem in accessing the link

**Comment by JasonYeMSFT** [comment]
@Shivamd1 Could you please verify and close this issue if it is fixed?

**Comment by Shivamd1** [comment]
Verified the bug in Version: 1.16.0-dev the issue does not reproduces so closing the bug
